File: src/app/models/Arrondissement.ts
import {Commune} from "./Commune";

export class Arrondissement {
  id: number;
  libelle:	string;
  createBy:	string;
  deleteBy:	string;
  commune: Commune;

  constructor(id: number, libelle: string, createBy: string, deleteBy: string, commune: Commune) {
    this.id = id;
    this.libelle = libelle;
    this.createBy = createBy;
    this.deleteBy = deleteBy;
    this.commune = commune;
  }
}

File: sgifd-backend/src/main/java/com/sgifdbackend/demo/parametrage/entites/Arrondissement.java
package com.sgifdbackend.demo.parametrage.entites;

import com.sgifdbackend.demo.entitiesBase.EntityBaseBean;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Table(name = "t_Arrondissement")
public class Arrondissement extends EntityBaseBean implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String libelle;

    @ManyToOne
    private Commune commune;

    public Arrondissement() {
    }

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getLibelle() {
		return libelle;
	}

	public void setLibelle(String libelle) {
		this.libelle = libelle;
	}

	public Commune getCommune() {
		return commune;
	}

	public void setCommune(Commune commune) {
		this.commune = commune;
	}

}
